frontend/src/app/constants/constants.d.ts

```diff
@@ -10,13 +10,581 @@ type Ethnicity = "hispanic" | "not_hispanic";
 type Gender = "male" | "female" | "other";
 type YesNo = "YES" | "NO";
 type Role = "STAFF" | "RESIDENT" | "STUDENT" | "VISITOR" | "";
+type TribalAffiliation =
```

Comment:
This is fun!

Reply:
![image](https://user-images.githubusercontent.com/18486822/113359969-1406aa00-9306-11eb-9658-d42833e3c992.png)

😬